src/containers/Tenants/Tenants.js

```diff
@@ -94,14 +86,15 @@ class Tenants extends React.Component {
 
         return (
             <div className={b('controls')}>
-                <TextInput
-                    className={b('search')}
-                    placeholder="Database name"
-                    value={searchQuery}
-                    onUpdate={handleSearchQuery}
-                    hasClear
-                    autoFocus
-                />
+                <div className={b('search')}>
```

Comment:
`TextInput` classname sometimes incorrectly applied, so I wrapped it with a `div` with `classname` instead